packages/client/src/locale/types/country.ts

```diff
@@ -16,6 +15,8 @@ export interface Country {
   cultures: string[];
   defaultSubfolder: string;
   defaultCulture: string;
+  states?: number[];
```

Comment:
The `Country` type is defined for the client `getCountries` which does not have the `states` and `cities` in its response. Those properties are added only after normalization from the `fetchCountryStates` and `fetchCountryCities` actions, so it makes sense to have those properties defined in a type exclusive to the redux package. You can add those properties to the `CountriesEntity` which is located in `packages/redux/src/entities/types/countries.types.ts`.